actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Rescue errors in `ActionController::ParamsWrapper#process_action`
```

Comment:
Since this is a bugfix for an unreleased feature, I don't think a changelog is needed.